### ISSUE TITLE
Turn off reporting for the Maven plugin?

### ISSUE THREAD

**Issue by Jurrie** [vmlens/vmlens#2]
Hello,

When I use vmlens to test my code, I need to wait a very long time for the reports to finish. In the general case, the unit tests are all ok, and I don't case about the reports. (They will not show any issues.) Is there a way to turn off report generation, and only have the reports generated when the unit tests fail? Or better yet, only when I want them to be generated?

With kind regards,

Jurrie

**Comment by ThomasKrieger** [comment]
No, not yet. But I can implement it in the next release. Does the following functionality fit:

A new maven configuration:  analyzeTrace true or false. Default true
When set to false the trace gets not analyzed and no reports get generated. This also means that no data races or deadlocks will be detected.

 Best regards,
Thomas

**Comment by Jurrie** [comment]
Hmm, I wasn't aware that this would mean no data races and deadlocks detection. For me, this would be a good fit, but it should be very clear for other users that setting `analyzeTrace` to `false` will give those implications. Maybe name the configuration variable something like `deadlockAndDataRaceDetection`?

**Comment by ThomasKrieger** [comment]
Good point, deadlockAndDataRaceDetection is better.

**Comment by ThomasKrieger** [comment]
I have implemented this in release 1.1.2: Set 
```xml
	<deadlockAndDataRaceDetection>false</deadlockAndDataRaceDetection>
```
in the config to disable the processing
